src/transport_ofi.c

```diff
@@ -655,14 +655,14 @@ int populate_mr_tables(void)
                                     &shmem_transport_ofi_target_heap_addrs[i],
                                     sizeof(uint8_t*));
             if (err) {
-                RAISE_WARN_STR("Error getting heap addr from runtime KVS");
+                RAISE_WARN_STR("Get of heap addr from runtime KVS failed");
```

Comment:
Should we change "addr" -> "address" just to unify with the other warning messages?